src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-gap;
+  width: widthRow($cell-width, 7, $cell-gap);
+  padding: 0 20px;
+
+  @each $day, $position in $seven-days {
+    &--start-day--#{$day} :first-child {
+      margin-left: $position;
+    }
+  }
+
+  @for $last-day from 28 through 31 {
+    &--month-length--#{$last-day} :nth-child(n + #{$last-day + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    @include square($cell-width);
+    flex-shrink: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    background-color: #eee;
+    border: 1px solid black;
+    cursor: pointer;
+    transition: all 0.5s ease;
+
+    @for $index from 1 through 31 {
+      &:nth-child(#{$index})::before {
+        content: "#{$index}";
+        font-family: Arial, serif;
+        font-size: 30px;
```

Comment:
[Checklist](https://github.com/mate-academy/layout_calendar/blob/master/checklist.md) 7ht point

Reply:
Move to the normalize.scss
```
font-family: Arial, serif;
font-size: 30px;
```